packages/did-provider-ebsi/src/types.ts

```diff
@@ -43,5 +41,19 @@ export interface ICreateIdentifierArgs {
   kms?: string
   alias?: string
   type?: EbsiDidSpecInfo
-  options?: IKeyOpts
+  options?: {
+    methodSpecificId?: string // method specific id for import
+    secp256k1Key?: IKeyOpts
+    secp256r1Key?: IKeyOpts
+  }
+}
+
+export interface ICreateIdentifierOpts {}
```

Comment:
What is this doing here? Should not be needed. Or at least properly list the actual options available

Reply:
Removed unused type